src/allotropy/allotrope/schemas/adm/solution-analyzer/REC/2024/03/CHANGE_NOTES.md

```diff
@@ -0,0 +1,3 @@
+Base schema: http://purl.allotrope.org/json-schemas/adm/solution-analyzer/REC/2024/03/solution-analyzer.schema
+
+This is a proposed new schema for solution-analyzer devices.
```

Comment:
Still missing change note for adding `Data System Document`?

Reply:
I forgot to update this notes, done.